practice-two/src/index.html

```diff
@@ -392,29 +407,70 @@ <h4 class="text-color-primary review__card-name">
           </ul>
       </div>
     </section>
+  <!-- END REVIEW SECTION -->
 
-    <!-- START CTA SECTION -->
-    <section class="text-color-secondary cta">
-      <div class="container cta__wrapper">
-        <hgroup class="cta__header">
-          <h2 class="heading-section">Start Selling Instantly</h2>
-          <p class="text-normal text-center">Start Selling With Your Pre-paid account. No deposit needed.</p>
-        </hgroup>
-        <form action="javascript:void(0)" class="cta__form">
-            <input type="email" name="email" placeholder="Email" class="form-input cta__input">
-            <button type="submit" class="btn btn-secondary btn--cta">Register</button>
-        </form>
+     <!-- START PRCING SECTION -->
+    <section class="pricing">
+      <div class="container pricing__wrapper">
+        <h2 class="heading-section text-color-secondary text-center">
+          Subscribing Plans
+        </h2>
+        <div class="pricing__cards">
+          <div class="pricing__card">
+            <hgroup class="pricing__card-heading pricing__card--freemium">
```

Comment:
BEM naming convention for modifier incorrect

Reply:
can you suggest to me some good class names?